.github/workflows/pre-release.yml

```diff
@@ -17,28 +18,44 @@ jobs:
       ee_version: ${{ steps.version-properties.outputs.ee_version }}
       oss_version: ${{ steps.version-properties.outputs.oss_version }}
       commit: ${{ steps.version-properties.outputs.commit }}
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
     steps:
       - name: Checkout repository
         uses: actions/checkout@v3
-      - name: Download Metabase ${{ matrix.edition }} artifact
-        run: |
-          declare -a curlHeaders=('-H' "Accept: application/vnd.github+json" '-H' "Authorization: Bearer $GITHUB_TOKEN" '-H' "X-GitHub-Api-Version: 2022-11-28")
+      - name: 'Download artifact'
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const fs = require('fs');
+
+            const allArtifacts = await github.rest.actions.listWorkflowRunArtifacts({
+               owner: context.repo.owner,
+               repo: context.repo.repo,
+               run_id: context.payload.workflow_run.id,
+            });
```

Comment:
I got this approach from the github docs and it seemed much easier to read than the curl approach

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#using-data-from-the-triggering-workflow